src/transformers/models/sam/convert_sam_original_to_hf_format.py

```diff
@@ -88,73 +133,71 @@ def replace_keys(state_dict):
     return model_state_dict
 
 
-def convert_sam_checkpoint(model_name, pytorch_dump_folder, push_to_hub, model_hub_id="ybelkada/segment-anything"):
-    checkpoint_path = hf_hub_download(model_hub_id, f"checkpoints/{model_name}.pth")
-
-    if "sam_vit_b" in model_name:
-        config = SamConfig()
-    elif "sam_vit_l" in model_name:
-        vision_config = SamVisionConfig(
-            hidden_size=1024,
-            num_hidden_layers=24,
-            num_attention_heads=16,
-            global_attn_indexes=[5, 11, 17, 23],
-        )
-
-        config = SamConfig(
-            vision_config=vision_config,
-        )
-    elif "sam_vit_h" in model_name:
-        vision_config = SamVisionConfig(
-            hidden_size=1280,
-            num_hidden_layers=32,
-            num_attention_heads=16,
-            global_attn_indexes=[7, 15, 23, 31],
-        )
+def convert_sam_checkpoint(model_name, pytorch_dump_folder, push_to_hub):
+    if model_name == "slimsam-50-uniform":
+        checkpoint_path = "/Users/nielsrogge/Documents/SlimSAM/SlimSAM-50-uniform.pth"
+    elif model_name == "slimsam-77-uniform":
+        checkpoint_path = "/Users/nielsrogge/Documents/SlimSAM/SlimSAM-77-uniform.pth"
```

Comment:
that's not very general

Reply:
let' s add a argument parsed for the path wdyt?

Reply:
Yes good idea